.github/workflows/test.sh

```diff
@@ -25,7 +25,7 @@ if [ "${maker}" = "make" ]; then
 fi
 if [ "${maker}" = "cmake" ]; then
     rm -rf build && mkdir build && cd build
-    cmake -DCMAKE_PREFIX_PATH=${top}/install/lib64/blaspp ..
+    cmake "-DCMAKE_PREFIX_PATH=${top}/install/lib64/blaspp" ..
```

Comment:
Using the `&&` just lets us put all the commands related to the `build` directory on one line.
This follows INSTALL.md (plus adding `rm`):
```
    mkdir build && cd build
    cmake ..
    make && make install
```
The `"..."` quotes on CMake's parameters match what is required in LAPACK++ test.sh
```
    cmake "-DCMAKE_PREFIX_PATH=${top}/install/lib64/blaspp;${top}/install/lib64/lapackpp" ..
```
so a comparison with `meld {blaspp,lapackpp}/.github/workflows/` highlights what was added.

Reply:
Ok, I now understand why we have `"..."`.

My point about the `&&` is if the script fails to go to the `build` repository (maybe due to `rm` or `mkdir`), it will call `cmake` from the wrong location and thus may generate an error.
I guess it is very unlikely to happen though.